usermanager/src/main/java/org/pesmypetcare/usermanager/clients/pet/PetManagerClient.java

```diff
@@ -92,18 +92,17 @@ public PetData getPet(String accessToken, String username,
 
     /**
      * Returns all the pets of the user.
+     *
      * @param accessToken The personal access token for the account
      * @param username The user's username
      * @return All the pets of the user
      * @throws ExecutionException When the retrieval of the pets fails
      * @throws InterruptedException When the retrieval is interrupted
      */
-    public List<Pet> getAllPets(String accessToken, String username) throws ExecutionException,
-            InterruptedException {
+    public List<Pet> getAllPets(String accessToken, String username) throws ExecutionException, InterruptedException {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [NCSS for this method is 17 (max allowed is 15).](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Libraries/pullRequest?prid=5561031)